### ISSUE TITLE
mac Catalina issue

### ISSUE THREAD

**Issue by sachinhr10** [BlueM/cliclick#91]
<img width="420" alt="Screenshot 2019-07-29 at 3 08 38 PM" src="https://user-images.githubusercontent.com/31371035/62042509-dfcb1400-b21b-11e9-80c6-4553b31afae5.png">

Getting above message when trying to run tests on mac Catalina machine. This machine is recently updated to mac 10.15. Cliclick was not mentioned in the list to grant access to.I tried giving full disk access to terminal as well, still got the same error message.

**Comment by sachinhr10** [comment]
When cliclick built locally and used, it says "permission denied" when we give permissions and run test cases, it again goes back to the initial permissions and ends up saying permission denied when tried to run next time.

**Comment by BlueM** [comment]
I just tried running cliclick (downloaded latest version from website, no local build) on a completely fresh 10.15 Beta installation. Upon 1st try (without having opened it in the Finder), I get the usual “This application cannot be opened because it’s from an unidentified and evil developer” complaint, as with every unsigned software. Then, after the usual countermeasure (right-click on application, and confirm that I really am sure I want to run the software), 2nd try: 10.15 asks me if Terminal should be granted Accessibility rights. And after I did that in system preferences, everything worked normally.
So I have no idea what’s the problem in your case, but it does not seem to be any general issue.
